src/Generator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Spatie\Honeypot;
+
+use Illuminate\Support\Str;
+
+class Generator
+{
+    public static function generate(): array
```

Comment:
Make this method non-static. Add a constructor where you pass the config values to.